include/unk_0204AEA8.h

```diff
@@ -0,0 +1,7 @@
+#ifndef POKEDIAMOND_UNK_0204AEA8_H
+#define POKEDIAMOND_UNK_0204AEA8_H
+
```

Comment:
usually a good idea to include global.h and the headers that define the structs here, so you don't have to worry about inlcuding that in the .c file